monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional, List, Dict
+from monitoring.monitorlib.typing import ImplicitDict
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
+from monitoring.uss_qualifier.common_data_definitions import Severity
+
+### Begin Definitions around users and test executor 
```

Comment:
None of the three classes in this section belong in a separate section from the others because they are all constitute the interface between SCD data generation and test executor -- if we look at a set of generated data that test executor can load, it will contain instances of all these types.

[This comment](https://github.com/interuss/dss/pull/669#discussion_r782463072) was because some test executor configuration parameters (`uss_base_url`) were being included in this file.  The simulator could not provide those inputs (instead, they would be provided at test executor runtime in the test configuration), so they needed to be strongly segregated.  It does not appear that we have that strong segregation need any more.

Reply:
Thanks for this, I have removed the comments separating the definitions